data/json/items/tool/cooking.json

```diff
@@ -146,6 +146,20 @@
     "max_charges": 200,
     "flags": [ "ALLOWS_REMOTE_USE" ]
   },
+  {
+    "id": "f_char_smoker",
+    "type": "TOOL",
+    "name": { "str": "smoking rack" },
+    "description": "This is a crafting_pseudo_item if you have it something is wrong.",
+    "material": [ "steel" ],
+    "symbol": ";",
+    "color": "light_gray",
+    "ammo": [ "charcoal" ],
+    "pocket_data": [ { "pocket_type": "MAGAZINE", "ammo_restriction": { "charcoal": 2000 } } ],
+    "sub": "char_smoker",
+    "max_charges": 2000,
+    "flags": [ "ALLOWS_REMOTE_USE" ]
```

Comment:
It should probably have `"PSEUDO"` flag too?

Reply:
Thanks on both :)

Reply:
it technically won't change anything, as furniture pseudo items add the pseudo flag in form_from_map